lib/main_common.pm

```diff
@@ -1145,7 +1146,7 @@ sub load_consoletests {
     if (is_opensuse || !is_staging && (check_var_array('SCC_ADDONS', 'asmm') || is_sle('15+') && !is_desktop)) {
         loadtest "console/salt";
     }
-    if (check_var('ARCH', 'x86_64')
+    if (is_x86_64
         || check_var('ARCH', 'i686')
```

Comment:
I wonder if we are still having tests running on i686 or i586, @DimStar77, @lnussel, @coolo  do you guys have a clue? :)

Reply:
@foursixnine, for Tumbleweed i586: https://openqa.opensuse.org/tests/overview?distri=microos&distri=opensuse&version=Tumbleweed&build=20190505&groupid=1

Reply:
Ah :( it's actually there for both